docs/content/docs/install/_index.md

```diff
@@ -91,4 +91,4 @@ $ helm get manifest osm --namespace osm-system
 
 ## Next Steps
 
-Now that the OSM control plane is up and running, [add services](../onboard_services/) to the mesh.
+Now that the OSM control plane is up and running, [add services](../tasks_usage/onboard_services.md) to the mesh.
```

Comment:
```suggestion
Now that the OSM control plane is up and running, [add services](../tasks_usage/onboard_services) to the mesh.
```

`.md` extension will not render on the docs.openservicemesh.io website.

Reply:
Thanks for mentioning this and I follow https://github.com/openservicemesh/osm/blob/release-v0.8/docs/README.md#internal-links and I can confirm all the link can be work on website.